test-data/src/test/resources/org/openmrs/module/fhir2/api/dao/impl/FhirPersonDaoImplTest_initial_data.xml

```diff
@@ -9,8 +9,8 @@
     graphic logo is a trademark of OpenMRS Inc.
 -->
 <dataset>
-    <person person_id="2" gender="M" birthdate="1999-12-20" dead="false" creator="1" date_created="2005-01-01 00:00:00.0" voided="false" uuid="61b38324-e2fd-4feb-95b7-9e9a2a4400df"/>
-    <person person_id="3" gender="M" birthdate="2000-11-18" dead="true" creator="1" date_created="2005-01-01 00:00:00.0" voided="false" uuid="5c521595-4e12-46b0-8248-b8f2d3697766"/>
+    <person person_id="2" gender="M" birthdate="1999-12-20" dead="false" creator="1" date_created="2005-01-01 00:00:00.0" voided="false"  uuid="61b38324-e2fd-4feb-95b7-9e9a2a4400df"/>
+    <person person_id="3" gender="M" birthdate="2000-11-18" dead="false" creator="1" date_created="2005-01-01 00:00:00.0" voided="false" void_reason="there is no reason" uuid="5c521595-4e12-46b0-8248-b8f2d3697766"/>
```

Comment:
Instead of changing this, please use the person with person_id=4

Reply:
@varung-31 thanks for the review though i see no problem with making the change if it doesn't break any code or tests and it works well with the test

Reply:
@gitcliff Yes, but why change it if your code can work without this change ?

Reply:
@varung-31  the `person_id=4` doesn't work to ,,thats why i used `person_id="3"`